chainer/training/extensions/graph_report.py

```diff
@@ -0,0 +1,141 @@
+from chainer import reporter
+import chainer.serializer as serializer_module
+from chainer.training import extension
+import chainer.training.trigger as trigger_module
+import json
+import numpy as np
+from os import path
```

Comment:
Could you move this line to the top of this file? The reason is ditto to the former lines (according to the contribution guide).